### PR TITLE
Adds Tremblay atmospheres

### PR DESCRIPTION
**Pull request by Raindogjones**

### CODE REVIEW

**Review by aprsa — APPROVED** [review]
This looks good to me. I want to relocate magic strings to the top of the module (so that we never actually refer to individual atmospheres anywhere in the code), but this needs to be done systematically and is outside of the scope for this PR.

**Comment by aprsa** [comment]
Are Tremblay tables also on Terra?

**Comment by Raindogjones** [comment]
Yep, the 4 TMAP grids and Tremblay are all tarballed in my home directory.

File: phoebe/frontend/default_bundles/default_binary.bundle [hunk 1]
```diff
@@ -426,9 +426,13 @@ null
...
 "choices": [
 "auto",
+"tmap_DA",
 "phoenix",
 "tmap_sdO",
-"ck2004"
+"tmap_DO",
+"ck2004",
+"tremblay",
+"tmap_DAO"
```
Comment (kecnry) [inline phoebe/frontend/default_bundles/default_binary.bundle:435]: were the tmap ones just overlooked in the past?

File: phoebe/frontend/default_bundles/default_contact_binary.bundle [hunk 2]
```diff
@@ -144,7 +144,7 @@ null
 "kind": "star",
 "context": "component",
 "description": "Critical (maximum) value of the equivalent radius for the given morphology",
-"value": 1.6724563972838287,
+"value": 1.6724563972838289,
```
Comment (kecnry) [inline phoebe/frontend/default_bundles/default_contact_binary.bundle:147]: I vaguely remember seeing updates here recently... do we understand why this is changing?

File: phoebe/frontend/default_bundles/default_binary.bundle (hunk 1 above)
Comment (Raindogjones) [reply phoebe/frontend/default_bundles/default_binary.bundle:435]: I think we just never rebuilt the default bundles in this particular branch.

**Pull request merged**